client/src/document/ingredients/browser-compatibility-table/utils.ts

```diff
@@ -191,3 +193,20 @@ export function getCurrentSupport(
   // Default (likely never reached)
   return getFirst(support);
 }
+
+export function getPreviousVersion(
+  version: BCD.VersionValue | string | undefined,
+  browser: BCD.BrowserStatement
+): BCD.VersionValue | string | undefined {
```

Comment:
`VersionValue` already includes `string` (and `null`, for that matter), and I don't think we'll ever get `undefined`:

```suggestion
export function getPreviousVersion(
  version: BCD.VersionValue,
  browser: BCD.BrowserStatement
): BCD.VersionValue {
```

Reply:
Done, though worth noting I copied that from here, so it may also be worth changing:
https://github.com/mdn/yari/blob/f59e4c8382648e65596e673f74bfe90bf0b20003/client/src/document/ingredients/browser-compatibility-table/utils.ts#L97-L100